R/derive_param_extreme_record.R

```diff
@@ -136,6 +136,16 @@ derive_param_extreme_record <- function(dataset = NULL,
                                         order,
                                         mode,
                                         set_values_to) {
+  deprecate_inform(
+    when = "1.2.0",
+    what = "derive_param_extreme_record()",
+    with = "derive_extreme_event()",
+    details = c(
+      x = "This message will turn into a warning with release of 1.3.0",
+      i = "https://pharmaverse.github.io/admiral/reference/derive_extreme_event.html"
```

Comment:
I think I may have made this suggestion previously, but I wouldn't promise to the users the precise version when we're going to make deprecation message changes. We have a long deprecation cycle, and who knows if we need to change things in the future, e.g. we release v1.3 earlier than we had previously planned, or we wait longer. I recommend that before each release (and not just after a release), we search for all the deprecation messages in our package and update based on the deprecation calendar.

Reply:
You did!  ummm.... maybe just "This message will turn into a warning at the next release"

I just want to stress to the user to start looking at updating their code.

Reply:
That could work, but also has a similar issue where if we promise the next release, but then the release timing changes for whatever reason. Also, we could decide to change our deprecation cycle.

My general advice is to not promise the users anything we don't have to. We have lengthy long deprecation cycle, which gives users literally years to update their code. I think the default messaging is sufficient. But, if you don't like it, you do you!!! 🕺🏼